content/SCALETutorials/Storage/Disks/ReplacingDisks.md

```diff
@@ -134,21 +133,21 @@ Click **Detach** on the **ZFS Info** widget on the **Devices** screen for the di
 Select **Confirm**, then click **Detach**.
 TrueNAS detaches the disk from the pool and promotes the hot spare disk to a full member of the pool.
 
-### Recreating the Hot Spare
+Refresh the screen to ensure the promoted hot spare appears in the pool as expected.
+
+#### Recreating a Hot Spare
 
 After promoting the hot spare, recreate the **Spare** vdev and assign a disk to it.
 
 {{< expand "Do I really need to promote the hot spare and then recreate the spare vdev?" "v" >}}
-If you have a hot spare inserted into the pool and then follow the instructions in [Replacing a Failed Disk](#replacing-a-failed-disk), TrueNAS automatically returns the hot spare disk to the existing **Spare** vdev and **ONLINE** status.
+If you have a hot spare inserted into the pool and then follow the instructions in [Replacing a Failed Disk Without a Hot Spare](#replacing-a-failed-disk-without-a-hot-spare), TrueNAS automatically returns the hot spare disk to the existing **Spare** vdev and **ONLINE** status.
 
 However, we do not recommend this method, because it causes two resilver events: one when activating the hot spare and again when replacing the failed disk.
 Resilvering degrades system performance until completed and causes unnecessary strain on the disk.
 
-To avoid unnecessary resilvers, [promote the hot spare](#detaching-a-failed-disk) then recreate the hot spare vdev.
+To avoid unnecessary resilvers, promote the hot spare by [detaching the failed disk]](#detaching-a-failed-disk) then recreate the hot spare vdev.
 {{< /expand >}}
 
-WHAT IF I DO NOT HAVE A HOT SPARE/A SYSTEM WHICH DOES NOT SUPPORT HOT SWAPPING? <<< Offline/reboot process described here >>>
```

Comment:
In which sequence should the steps be completed for the user if their system does not allow hot plug or hot swapping drives?